### PR TITLE
Tidied the Install from ansible file. Added Notice for AKSK and Profile Creation from Dashboard

### PR DESCRIPTION
**Pull request by anvithks**
**What type of PR is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespace from that line:
/kind documentation
/kind tidy

**What this PR does / why we need it**:
Updated the Install from ansible page with mandatory notice for creating AK/SK and Profile from dashboard.
Tidied up the page. Fixed some markdown issues.

**Test Report Added?**:
/kind TESTED
> /kind NOT-TESTED

**Test Report**:

**Special notes for your reviewer**:

### CODE REVIEW

File: content/soda-gettingstarted/installation-using-ansible.md [hunk 1]
```diff
@@ -241,51 +253,70 @@ Delete the Fileshare
...
+### Please Note
+{{% notice info %}}
+**_To use the multicloud service, an AK/SK must be generated before anything else. To do this you can follow the steps below:_**  <br />
+1. Go to AK/SK Management<br />
+2. Click on Add AK/SK button.<br />
+3. Save the file (Do not forget to save this file and keep it safe.)<br />
+{{% /notice %}}
+
+{{% notice info %}}
+**_To use the Block and File service a profile must be created before anything else. To do this you can follow the steps below:_**  <br />
```
Comment (kumarashit) [inline content/soda-gettingstarted/installation-using-ansible.md:270]:
```suggestion
**_To use the Block and File service, respective profile must be created before using these services. To do this you can follow the steps below:_**  <br />
```

File: content/soda-gettingstarted/installation-using-ansible.md [hunk 2]
```diff
@@ -241,51 +253,70 @@ Delete the Fileshare
...
+1. Go to Profiles<br />
+2. Click on Create button.<br />
+3. Enter the details and select the Storage type as Block or File. Create one profile for Block type and one for File.<br />
```
Comment (kumarashit) [inline content/soda-gettingstarted/installation-using-ansible.md:273]: Please mention the profile names must be default_block or default_file for block and file, respectively

File: content/soda-gettingstarted/installation-using-ansible.md [hunk 3]
```diff
@@ -1,3 +1,9 @@
+---
+title: Installation Guide using Ansible
+description: "This document describes how to install SODA projects in a local cluster with detailed configuraiton options. These steps will help you to enable / disable projects. After installation using these steps, you can get the features of all the enabled projects. You can test either through SODA Dashboard UI or CLI"
```
Comment (kumarashit) [inline content/soda-gettingstarted/installation-using-ansible.md:3]: This text is duplicated

File: content/soda-gettingstarted/installation-using-ansible.md (hunk 2 above)
Comment (anvithks) [reply content/soda-gettingstarted/installation-using-ansible.md:273]: Will make the change.

File: content/soda-gettingstarted/installation-using-ansible.md (hunk 3 above)
Comment (anvithks) [reply content/soda-gettingstarted/installation-using-ansible.md:3]: This description is not visible on the page. It is the front matter and this description is used to display the description on the Getting Started page.

Comment (kumarashit) [reply content/soda-gettingstarted/installation-using-ansible.md:3]: ok

**Review by kumarashit — APPROVED** [review]
LGTM

**Pull request merged**